### PR TITLE
Final updates before releasing 1.0 to PyPI

### PR DESCRIPTION
**Pull request by mmjb**

### CODE REVIEW

**Comment by jackson-flux** [comment]
Looks good to me!

File: setup.py [hunk 1]
```diff
@@ -5,8 +5,11 @@
 
 setuptools.setup(
     name="tf2_gnn",
-    version="0.9.21",
+    version="1.0.0",
+    author="Marc Brockschmidt",
+    author_email="mabrocks@microsoft.com",
```
Comment (pashminacameron) [inline setup.py]: Email is different from the one in the author list, but it doesn't matter!

Comment (pashminacameron) [reply setup.py]: Aliases, and all the fun. :)

File: setup.py [hunk 2]
```diff
@@ -2,16 +2,29 @@
...
     author_email="mabrocks@microsoft.com",
     description="TensorFlow 2.0 implementation of Graph Neural Networks.",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
     url="https://github.com/microsoft/tf2-gnn/",
     python_requires=">=3.6",
-    install_requires=["numpy", "docopt", "dpu-utils>=0.2.7", "h5py"],
+    install_requires=[
+        "numpy",
+        "tensorflow>=2.0.0",
```
Comment (pashminacameron) [inline setup.py]: :)

File: pipelines/azure-ci.yml [hunk 3]
```diff
@@ -1,25 +1,33 @@
-# Test TF2-GNN package
+# Test TF2-GNN package and build sdist to be consumed by release pipeline
 trigger:
   branches:
     include:
-      - '*'  # must quote since "*" is a YAML reserved character; we want a string
```
Comment (pashminacameron) [inline pipelines/azure-ci.yml]: Maybe the Github diff interface isn't super-helpful here, but it looks like publish will get triggered off other branches as well. Am I missing something?

Comment (mmjb) [reply pipelines/azure-ci.yml]: Publish just leaves an artifact in the pipeline-associated storage, upload to PyPI still requires a manual trigger in Azure DevOps interface.

**Pull request merged**